src/convert.ts

```diff
@@ -351,7 +382,11 @@ export function convertShare(json: unknown): ShareResult {
     });
   }
 
-  v8InitSource.workbench = workbenchIds.reverse();
+  v8InitSource.workbench = workbenchIds
+    .sort((a, b) =>
+      typeof a.index !== "undefined" ? a.index - (b.index ?? a.index + 1) : 1
+    )
+    .map((item) => item.id);
```

Comment:
Sort workbenchIds so items without `nowViewingIndex` will be at the end of the workbench